### PR TITLE
Feature/update-footer-order-align

### PR DESCRIPTION
**Pull request by ozanisgor**
@ryancwalsh according to https://github.com/NEAR-Edu/near-certification-tools/pull/7#issuecomment-1041509218 I did below changes.

1. Update footer content order to `Logo > Icons > Jobs`
2. Align footer left (In this point I am not sure did I understand correctly)
   - For this change I remove the space at left and right for the entire footer but I might be misunderstood so because of that I did an alternative version which align the entire footer to left and creates a space at right. 

   
17 02 22_layout_before

17 02 22_layout_after

### **2nd Version**

17 02 22_layout_after_v2

### CODE REVIEW

**Review by ryancwalsh — CHANGES_REQUESTED** [review]
What I meant is to have all content of the footer left-aligned, without using 3 columns anymore.

More like this: 
image

But after you condense down to 1 "column" like that, you might want to experiment with centering the whole footer to see whether it looks better than the left-aligned example I just showed here.

**Comment by ozanisgor** [comment]
@ryancwalsh I think the left-align version is looking better.  This one was challanging because of responsivenes. It looks good but when you change to small screen logo getting smaller than other items. I tried to figure out but could not find the solution.

I tried to show what i mean by loom video:
https://www.loom.com/share/2633635447434b538e4153c951d24367

18 02 22_footer-align-left_after

PREVIOUS versions

18 02 22_footer-align-left_certered-version


17 02 22_layout_after

**Comment by ryancwalsh** [comment]
@ozanisgor I just made this video for you and then afterwards saw your video here and comments.

https://www.loom.com/share/d728ccf6b9084378a2dac4ae461d89c8

Most of my video is still what I wanted to say, but now that I see your helpful responsiveness demo, I think (unlike what I said in the video) it does make sense to have an inline-block around the social icons. 

In the video I'd said that you could optionally have no element to contain them and that they could exist as direct siblings of the other part of the footer. 

But I think having them bundled together makes sense. But not acting as a column or with padding.

**Comment by ozanisgor** [comment]
@ryancwalsh thank you for the video to explain detailed. I hope I understand what you mean. I remove the unnecessary divs and tried to bring all content into one column. While doing this tried to keep responsiveness. I tried to use `inline-block` but somehow it doesn't work or I coundn't manage to make that work but with flex(according to VSCode they are doing the same) I belive now seems like what you want to see. I tried to explain it with a video and some screenshots below.

I didn't push the code because maybe this not what you want. I can push immedately if you want.

BEFORE

before_footer-align-left

AFTER

after_footer-align-left

AFTER-Centered 

after_footer-align-center

AFTER-Small-Devices

after_footer-align-center_small-devices

AFTER-Small-Devices with `inline-block`

Also wider screens have the same wierd looking.

after_footer-align-center_small-devices_inline-block

VIDEO

When I replay the video I've tnoticed voice is lagging I hope It won't be a problem for you to follow.

https://www.loom.com/share/97cb91d034fb494bba9c6a926b14d8a3

**Comment by ozanisgor** [comment]
@ryancwalsh sorry for the problem with the video. I pushed the code as `centered` one. 
I didn't push at the first place because maybe I misunderstood what you want and didn't want to mess.

File: web-app/components/Footer.tsx [hunk 1]
```diff
@@ -1,28 +1,16 @@
...
-        <div className="flex justify-center space-x-6 pb-12 md:pb-0">
-          <div className="mt-8 md:mt-0 md:order-1">
-            <a href="https://NEAR.University" target="_blank" rel="noopener noreferrer">
-              <img alt="NEAR University logo" width="200px" src="/near_university_logo.svg" />{' '}
-            </a>
-          </div>
-        </div>
+      <div className="mx-auto px-4 flex items-center justify-center">
+        <a className="pt-5 mr-8 pb-5" href="https://NEAR.University" target="_blank" rel="noopener noreferrer">
+          <img className="min-w-full" alt="NEAR University logo" width="200px" src="/near_university_logo.svg" />{' '}
```
Comment (ryancwalsh) [inline web-app/components/Footer.tsx:8]: This `{' '}` feels weird. Let's fix it in a future branch.

**Pull request merged**